plugins/woocommerce-blocks/assets/js/blocks/collection-filters/inner-blocks/attribute-filter/frontend.ts

```diff
@@ -50,10 +50,10 @@ store( 'woocommerce/collection-attribute-filter', {
 
 			const context = getContext< AttributeFilterContext >();
 
-			if ( dropdownContext.selectedItem.value ) {
+			if ( dropdownContext.selectedItems[ 0 ].value ) {
 				navigate(
 					getUrl(
-						[ dropdownContext.selectedItem.value ],
+						[ dropdownContext.selectedItems[ 0 ].value ],
```

Comment:
Looks like this bug was hanging around from when I converted dropdown to support multiple items.